.github/workflows/ci.yml

```diff
@@ -1,5 +1,9 @@
 name: CI
 
+concurrency:
```

Comment:
```
concurrency:
  group: ${{ github.workflow }}-${{ github.ref }}
  cancel-in-progress: true
```

shouldn't we have separate queues for PR benchmarks and ci runs?

https://docs.github.com/en/enterprise-cloud@latest/actions/writing-workflows/choosing-what-your-workflow-does/control-the-concurrency-of-workflows-and-jobs#example-using-concurrency-and-the-default-behavior